Makefile

```diff
@@ -612,7 +612,7 @@ verify: modules ## Run all verification scripts
 verify: ## Run all verification scripts
 	$(MAKE) smoke-build generate-go generate
 	./packages/tkg-clusterclass/hack/sync-cc.sh
-	./hack/verify-dirty.sh
+	#./hack/verify-dirty.sh # TODO: uncomment after testing
```

Comment:
What is requiring the skipping of this ATM?

Reply:
reverted the change. commented this to run tests. 
verify step is failing need to investigate.